navigation/create_control/config/create_2_control.yaml

```
@@ -0,0 +1,43 @@
+# http://wiki.ros.org/diff_drive_controller for more parameters.
+
+mobile_base_controller:
+  type        : "diff_drive_controller/DiffDriveController"
+  left_wheel  : 'wheel_left_joint'
+  right_wheel : 'wheel_right_joint'
+  publish_rate: 50.0               # default: 50
```

Gazebo and real robot run at 30 Hz.

```suggestion
  publish_rate: 30.0               # default: 50
```

---

done